### ISSUE TITLE
Upgrading a tier 2 launch pad resets the pad to tier 1

### ISSUE THREAD

**Issue by deimos790** [magico13/KCT#130]
Title; might just be cosmetic, and might just last till it's upgraded, but facility now says (and seems to apply the limits of ) level 1. (1.0.5)
[Log](https://www.dropbox.com/s/t7b2up78l7sagit/Player%20%28KCT%29.log?dl=0)

**Comment by setimike** [comment]
Yes, it upgrades visually immediately, but then changes back to level one appearance until upgrade is complete. So appearance is accurate.
